harper-core/src/linting/closed_compounds.rs

```diff
@@ -82,12 +82,94 @@ create_closed_compound_linter!(Northeastern, "north eastern", "northeastern");
 create_closed_compound_linter!(Upholstery, "up holstery", "upholstery");
 create_closed_compound_linter!(Proofread, "proof read", "proofread");
 
+create_closed_compound_linter!(Forever, "for ever", "forever");
+create_closed_compound_linter!(Into, "in to", "into");
+create_closed_compound_linter!(Everyone, "every one", "everyone");
+create_closed_compound_linter!(Nobody, "no body", "nobody");
+create_closed_compound_linter!(Somebody, "some body", "somebody");
+create_closed_compound_linter!(Anybody, "any body", "anybody");
+create_closed_compound_linter!(Nothing, "no thing", "nothing");
+create_closed_compound_linter!(Anyway, "any way", "anyway");
+create_closed_compound_linter!(Anytime, "any time", "anytime");
+create_closed_compound_linter!(Anywhere, "any where", "anywhere");
+create_closed_compound_linter!(Nowhere, "no where", "nowhere");
+create_closed_compound_linter!(Everywhere, "every where", "everywhere");
+create_closed_compound_linter!(Overnight, "over night", "overnight");
+create_closed_compound_linter!(Instead, "in stead", "instead");
+create_closed_compound_linter!(Backpack, "back pack", "backpack");
+create_closed_compound_linter!(Cupboard, "cup board", "cupboard");
+create_closed_compound_linter!(Keyboard, "key board", "keyboard");
+create_closed_compound_linter!(Somewhere, "some where", "somewhere");
+create_closed_compound_linter!(Notebook, "note book", "notebook");
+create_closed_compound_linter!(Middleware, "middle ware", "middleware");
+create_closed_compound_linter!(Firmware, "firm ware", "firmware");
+create_closed_compound_linter!(Smartwatch, "smart watch", "smartwatch");
+create_closed_compound_linter!(Touchscreen, "touch screen", "touchscreen");
+create_closed_compound_linter!(Headset, "head set", "headset");
+create_closed_compound_linter!(Framework, "frame work", "framework");
+create_closed_compound_linter!(Touchpad, "touch pad", "touchpad");
+create_closed_compound_linter!(Microprocessor, "micro processor", "microprocessor");
+create_closed_compound_linter!(Datacenter, "data center", "datacenter");
+create_closed_compound_linter!(Smartphone, "smart phone", "smartphone");
+create_closed_compound_linter!(Webcam, "web cam", "webcam");
+create_closed_compound_linter!(Headphone, "head phone", "headphone");
+create_closed_compound_linter!(Desktop, "desk top", "desktop");
+create_closed_compound_linter!(Laptop, "lap top", "laptop");
+create_closed_compound_linter!(Overclocking, "over clocking", "overclocking");
+create_closed_compound_linter!(Backplane, "back plane", "backplane");
+create_closed_compound_linter!(Smartcard, "smart card", "smartcard");
+create_closed_compound_linter!(Bitrate, "bit rate", "bitrate");
+create_closed_compound_linter!(Overload, "over load", "overload");
+create_closed_compound_linter!(Underclock, "under clock", "underclock");
+create_closed_compound_linter!(Blockchain, "block chain", "blockchain");
+create_closed_compound_linter!(Opensource, "open source", "opensource");
```

Comment:
"open source" is preferred over "opensource" which even my browser flags. 
<img width="219" alt="image" src="https://github.com/user-attachments/assets/210c6ff8-0597-49ad-9b79-65c67bbb345a" />